wdl/transforms/draft3/src/main/scala/wdl/draft3/transforms/ast2wdlom/package.scala

```diff
@@ -1,24 +1,44 @@
 package wdl.draft3.transforms
 
-import cats.syntax.validated._
-import common.validation.ErrorOr.ErrorOr
-import wdl.draft3.parser.WdlParser.{AstNode, Terminal}
+import better.files.File
+import cats.data.Kleisli
+import cats.instances.either._
+import common.Checked
+import common.validation.Checked._
+import wdl.draft3.parser.WdlParser.{Ast, AstNode, Terminal}
+import wdl.draft3.transforms.ast2wdlom.CheckedAstNodeToAst.CheckedAstNodeToAst
 import wdl.draft3.transforms.parsing.FileParser
+import wdl.model.draft3.elements.FileElement
 
 package object ast2wdlom {
 
-  implicit val astFromAstNode = AstFromAstNode
-  implicit val draft3FileElementFromAstNode = FromAtoB.viaX(AstFromAstNode, Draft3FileElementFromAst)
-  implicit val draft3ImportElementFromAstNode = FromAtoB.viaX(AstFromAstNode, Draft3ImportElementFromAst)
-  implicit val draft3TaskDefinitionElementFromAstNode = FromAtoB.viaX(AstFromAstNode, Draft3TaskDefinitionElementFromAst)
-  implicit val draft3WorkflowDefinitionElementFromAstNode = FromAtoB.viaX(AstFromAstNode, Draft3WorkflowDefinitionElementFromAst)
+  type CheckedAtoB[A, B] = Kleisli[Checked, A, B]
+  object CheckedAtoB {
+    def apply[A, B](implicit runner: CheckedAtoB[A, B]): Kleisli[Checked, A, B] = runner
+    def apply[A, B](run: A => Checked[B]): Kleisli[Checked, A, B] = Kleisli(run)
```

Comment:
ToL might as well declare `def apply[A, B](run: A => ErrorOr[B]): Kleisli[Checked, A, B] = Kleisli(run.map(_.toEither)`, save yourself conversions elsewhere

Reply:
Bad news:
```
Error:(11, 9) double definition:
def apply[A, B](run: A => common.Checked[B]): cats.data.Kleisli[common.Checked,A,B] at line 10 and
def apply[A, B](run: A => common.validation.ErrorOr.ErrorOr[B]): cats.data.Kleisli[common.Checked,A,B] at line 11
have same type after erasure: (run: Function1)cats.data.Kleisli
    def apply[A, B](run: A => ErrorOr[B]): Kleisli[Checked, A, B] = Kleisli(runThenCheck(run))
```

Reply:
so I can get around this by giving the apply's real names. A bit sad that I had to, but whatchagonnado...